tools/lib/h5tools_dump.c

```diff
@@ -2220,10 +2220,6 @@ h5tools_print_datatype(FILE *stream, h5tools_str_t *buffer, const h5tool_format_
                 h5tools_str_append(buffer, "H5T_NATIVE_FLOAT");
             else if (H5Tequal(type, H5T_NATIVE_DOUBLE) == TRUE)
                 h5tools_str_append(buffer, "H5T_NATIVE_DOUBLE");
-#if H5_SIZEOF_LONG_DOUBLE != 0
```

Comment:
Why is this removed?   Seems like 128-bit long doubles should be output.

Reply:
#if H5_SIZEOF_LONG_DOUBLE != 0 is removed because it is always true on the machines we have tested (also see #470 discussion).
The block is removed because of these reasons, it is not cross-platform, there is no test to determine which platform has the correct size (i.e. windows defines long double as the same size as double), and 128-bit [little|big]-endian float is consistently output in tools tests on all our test machines.

Reply:
I've thought about this a little more, and don't think I was clear earlier:  I understand that H5_SIZEOF_LONG_DOUBLE is >0 always, but what I was getting at was that we define H5T_NATIVE_LDOUBLE, so if H5Tequal(type, H5T_NATIVE_LDOUBLE) is true, then we should print "H5T_NATIVE_LDOUBLE", like we do for all other native types we predefine.   If we don't/can't do that, that I believe that H5T_NATIVE_LDOUBLE should be removed from the public header file.